src/transformers/convert_graph_to_onnx.py

```diff
@@ -214,25 +320,60 @@ def convert(
         convert_tensorflow(nlp, opset, output)
 
 
-def verify(path: str):
+def quantize(onnx_model_path: Path) -> Path:
+    """
+    Quantize the weights of the model from float32 to in8 to allow very efficient inference on modern CPU.
+    Args:
+        onnx_model_path: Path to location the exported ONNX model is stored
+
+    Returns: The Path generated for the quantized
+    """
+
+    try:
+        ensure_onnxruntime_installed()
+        import onnx
+        from onnxruntime import __version__ as ort_version
+        from onnxruntime.quantization import quantize, QuantizationMode
+
+        print(f"Found ONNX: {onnx.__version__}")
+        print(f"Found ONNXRuntime: {ort_version}")
+
+        onnx_model = onnx.load(onnx_model_path.as_posix())
```

Comment:
Thanks Morgan for integrating onnx quantization! For transformer-based models, it gets better performance to  optimize the model before quantizing:

```
from onnxruntime import SessionOptions, InferenceSession

# optimize the model and save it
opt_model_path = generate_identified_filename(onnx_model_path, "-opt")
sess_option = SessionOptions()
sess_option.optimized_model_filepath = opt_model_path
sess = InferenceSession(onnx_model_path, sess_option)

# load and quantize optimized model
onnx_model = onnx.load(opt_model_path.as_posix())
quantized_model = quantize(
            model=onnx_model, quantization_mode=QuantizationMode.IntegerOps, force_fusions=True, symmetric_weight=True,
)

# Save model
print(f"Storing quantized model at {quantized_model_path}")
onnx.save(quantized_model, quantized_model_path.as_posix())

```

Reply:
@yufenglee I've a dev branch which will bring the optimisations you mentioned into the export script. I decided to keep them in separate PR for easier reviewing 😃. 

Should be in before the end of the week 👍

Reply:
How actually it's exactly the same way of optimizing as I was following the content of the notebook you included upstream [in ONNX runtime](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb).

Which way would be better @yufenglee ? 

```python
# optimize transformer-based models with onnxruntime-tools
from onnxruntime_tools import optimizer
from onnxruntime_tools.transformers.onnx_model_bert import BertOptimizationOptions

# disable embedding layer norm optimization for better model size reduction
opt_options = BertOptimizationOptions('bert')
opt_options.enable_embed_layer_norm = False

opt_model = optimizer.optimize_model(
    'bert.onnx',
    'bert', 
    num_heads=12,
    hidden_size=768,
    optimization_options=opt_options)
opt_model.save_model_to_file('bert.opt.onnx')
```

Reply:
@mfuntowicz, onnxruntime_tools is more flexible, for example, you can enable/disable certain optimization.